DabraHalleloEndaAbunaSamuel/DHAS022.xml

```diff
@@ -0,0 +1,150 @@
+                                       </support>
+                                       <extent>                                          
+                                                       <measure unit="leaf" quantity="40">40</measure>                                  
+                                                       <dimensions type="outer" unit="mm">
+                                                           <height cert="low">130</height>
+                                                           <width cert="low">110</width>
+                                                           <depth cert="low">30</depth>
+                                                           
+                                           </dimensions>
+                                       </extent>   
+                            <condition key="deficient">Approximately a 
+                                half of the manuscript, with back board, has been preserved.</condition>
+                                                     
+                         </supportDesc>            
+                                                  
+                         <layoutDesc>
+                             <layout columns="2">                                 
+                             </layout>
+                         </layoutDesc>                       
+                         
+                         
+                     </objectDesc>
+                       
+                    
+                           
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script" cert="low">17th-century script.</seg>                                                        
+                           </handNote>                           
+                       </handDesc>   
+                       
+                           
+                       <decoDesc>
+                           <decoNote type="band" xml:id="d11">
+                               <desc>Short bichrome (red, black) ornamental bands at the beginning of some text parts.</desc>
+                           </decoNote>
+                       </decoDesc>
+                       
+                       <additions>
+                       <list>
+                           
+                           <item xml:id="a1">                           
+                           <desc type="Correction">Correction on some pages 
+                               (omissions added interlinerally in the main hand)</desc>                          
+                               </item>
+                           
+                           <item xml:id="a2"> 
+                               <locus from="40rb" to="40va"/>
+                               <desc>An excerpt from a hymn to St Walatta Ṗeṭros</desc>                          
```

Comment:
Maybe it might be worth adding a reference to PRS9848Walatta to make it better findable

Reply:
Then, I would have to elaborate it more precisely (but there is only one Walatta Ṗeṭros :)).

Reply:
Ok! :) Later then.